### ISSUE TITLE
ReferenceInput filter doesn't work  when value is a Boolean

### ISSUE THREAD

**Issue by fdiogoc** [benwinding/react-admin-firebase#27]
```
      <ReferenceInput
        source="participanteId"
        reference="participantes"
        validate={required()}
        filter={{ isAdmin: true }}
      >
```
error in console:
`Params.filter[key].includes is not a function`

**Comment by benwinding** [comment]
Hi @fdiogoc,
Thanks for reporting this, I'll check it out!

**Comment by benwinding** [comment]
This should be fixed in version `0.6.3`. Assumed all input would be a string, now I know!
Let me know if there's any other issues :+1:
